features/navbar.feature

```diff
@@ -54,3 +54,18 @@ Scenario: visiting the events page from a different page
     Given I am on the people page
     Then I should see Contact
     And I should not see Contact Us
+
+  Scenario: Seeing account dropdown from homepage
+    Given I am on the home page
+    And I am logged in
+    Then I should see Logged in as
+    And I should not see Edit profile
+    And I should not see Log Out
+    When I follow Logged in as
```

Comment:
Let's ask Sophia about that; she didn't specify but hover would be cool. Clicking links seems like the most straightforward right now, though.